### PR TITLE
Polish README.md.

### PR DESCRIPTION
**Pull request by concretevitamin**
TODO
- [x] update docs once finalized

Tested (run the relevant ones):

- [ ] Code formatting: `bash format.sh`
- [ ] Any manual or new tests for this PR (please specify below)
- [ ] All smoke tests: `pytest tests/test_smoke.py` 
- [ ] Relevant individual smoke tests: `pytest tests/test_smoke.py::test_fill_in_the_name` 
- [ ] Backward compatibility tests: `bash tests/backward_comaptibility_tests.sh`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -71,7 +71,7 @@ You can find our documentation [here](https://skypilot.readthedocs.io/en/latest/
 - [Quickstart](https://skypilot.readthedocs.io/en/latest/getting-started/quickstart.html)
 - [CLI reference](https://skypilot.readthedocs.io/en/latest/reference/cli.html)
 
-## SkyPilot in 1 minute
+## SkyPilot in 1 Minute
```
Comment (romilbhardwaj) [inline README.md:74]: ~Super nit, feel free to ignore: Our capitalization is inconsistent in the readme. Some headlines (`Learn more`, `Issues, feature requests, and questions`) use [AP style](https://apstylebook.com/ask_the_editor_faq)(only proper nouns and 1st letter in caps), others (`Getting Started`) use CMS style (sentence case except for prepositions). 

EDIT - saw you changed the rest of the headlines too. Thanks for making the capitalization consistent!

**Review by romilbhardwaj — APPROVED** [review]
Thanks @concretevitamin!

**Review by Michaelvll — APPROVED** [review]
Thanks for the update @concretevitamin! LGTM.

File: README.md [hunk 2]
```diff
@@ -125,20 +125,31 @@ SkyPilot then performs the heavy-lifting for you, including:
...
+- Framework examples: [PyTorch DDP](https://github.com/skypilot-org/skypilot/blob/master/examples/resnet_distributed_torch.yaml), [DeepSpeed](./examples/deepspeed-multinode/sky.yaml), [JAX/Flax on TPU](https://github.com/skypilot-org/skypilot/blob/master/examples/tpu/tpuvm_mnist.yaml), [Stable Diffusion](https://github.com/skypilot-org/skypilot/tree/master/examples/stable_diffusion), [Detectron2](https://github.com/skypilot-org/skypilot/blob/master/examples/detectron2_docker.yaml), [Distributed](https://github.com/skypilot-org/skypilot/blob/master/examples/resnet_distributed_tf_app.py) [TensorFlow](https://github.com/skypilot-org/skypilot/blob/master/examples/resnet_app_storage.yaml), [programmatic grid search](https://github.com/skypilot-org/skypilot/blob/master/examples/huggingface_glue_imdb_grid_search_app.py), [Docker](https://github.com/skypilot-org/skypilot/blob/master/examples/docker/echo_app.yaml), and [many more (`examples/`)](./examples).
+
+Follow updates:
+- [Twitter](https://twitter.com/skypilot_org)
+- [Slack](http://slack.skypilot.co)
+- [SkyPilot Blog](https://blog.skypilot.co/) ([Introductory blog post](https://blog.skypilot.co/introducing-skypilot/))
```
[the remark this answers is not in the document]
Comment (concretevitamin) [reply README.md:145]: Thanks. Let me play with it later. IIRC I tried before and more buttons may make them hard to distinguish from each other.

**Pull request merged**